examples/streaming_collector/random_stream.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python
+
+# http://www.apache.org/licenses/LICENSE-2.0.txt
+#
+# Copyright 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import random
+import time
+
+import snap_plugin.v1 as snap
+
+LOG = logging.getLogger(__name__)
+
+
+class RandomStream(snap.StreamCollector):
+    """Rand
+
+    Streams random int and float metrics
+    """
+
+    def stream(self):
+        LOG.debug("Metrics streaming started")
+        while True:
+            time.sleep(1)
+            metrics = []
+            metric = snap.Metric(
+                namespace=[
+                    snap.NamespaceElement(value="intel"),
+                    snap.NamespaceElement(value="streaming"),
+                    snap.NamespaceElement(value="random"),
+                    snap.NamespaceElement(value="int")
+                ],
+                version=1,
+                tags={"mtype": "gauge"},
+                description="Random {}".format("int"),
```

Comment:
What's the purpose of this `format()` call? If addressed, it should also be changed in docs example.